CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+<a name="unreleased"></a>
+# unreleased
+
+### Feature
+* **terra-download-helper** added new optional parameter (boolean) for downloadFileFromUrl and set target='_blank' to link if parameter is true  
```

Comment:
**terra-download-helper** added new optional parameter for `downloadFileFromUrl` to open the file in new tab